<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Supriya" failures="1" tests="9" name="CARLoan Test" time="0.187" errors="0" timestamp="2024-12-13T19:46:00 EST">
  <testcase classname="TestP.day1" name="Login" time="0.007"/>
  <testcase classname="TestP.day1" name="RIDS" time="0.001"/>
  <testcase classname="TestP.day1" name="SIgnOut" time="0.002"/>
  <testcase classname="TestP.day1" name="SIgnOut" time="0.002"/>
  <testcase classname="TestP.day1" name="SIgnOut" time="0.001"/>
  <testcase classname="TestP.day1" name="plano" time="0.03">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at TestP.day1.plano(day1.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:840)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- plano -->
  <testcase classname="TestP.day4" name="Demo" time="0.001"/>
  <testcase classname="TestP.day4" name="SecondDemo" time="0.001"/>
  <testcase classname="TestP.day1" name="Gerland" time="0.0">
    <skipped/>
  </testcase> <!-- Gerland -->
</testsuite> <!-- CARLoan Test -->
